client/signup/steps/jpo-site-type/select-business-personal.jsx

```diff
@@ -1,55 +1,68 @@
+/**
+ * External dependencies
+ */
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import { translate } from 'i18n-calypso';
+import get from 'lodash/get';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
 import Card from 'components/card';
 import Button from 'components/button';
-
 import PersonalGraphic from './personal-graphic';
 import BusinessGraphic from './business-graphic';
 
-module.exports = React.createClass( {
+class SelectBusinessPersonal extends React.Component {
 
-	displayName: 'SelectBusinessPersonal',
+	static propTypes = {
+		signupDependencies: PropTypes.object,
+		required: PropTypes.bool
+	};
 
-	propTypes: {
-		required: React.PropTypes.bool,
-	},
-
-	render: function() {
+	render() {
 		if ( ! this.props.current ) {
 			return ( <div /> );
 		}
 
 		return ( 
-			<div className="jpo__site-type-wrapper">
-				<div className="jpo__site-type-row jpo__site-type-row-small">
-					<div className="card design-type-with-store__choice">
-						<a className="design-type-with-store__choice-link:after" href="#"
-						onClick={ this.props.onSelectPersonal }>
-						<div className="design-type-with-store__image">
-						<PersonalGraphic />
-						</div>
-						<div className="design-type-with-store__choice-copy">
-						<span className="button is-compact design-type-with-store__cta"
-						onClick={ this.props.onSelectPersonal }>{ translate( 'Personal site' ) }</span>
-						</div>
+			<div className="jpo-site-type__wrapper">
+				<div className="jpo-site-type__row jpo-site-type__row-small">
+					<Card className={ classNames( 'jpo-site-type__choice', {
+						'is-selected': 'personal' === get( this.props.signupDependencies, [ 'jpoSiteType', 'businessPersonal' ], '' )
+					} ) }>
+						<a className="jpo-site-type__choice-link" href="#" onClick={ this.props.onSelectPersonal }>
+							<div className="jpo-site-type__image">
+								<PersonalGraphic />
+							</div>
+							<div className="jpo-site-type__choice-copy">
+								<Button onClick={ this.props.onSelectPersonal }>
+									{ translate( 'Personal site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Personal Site' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).